src/main/java/wolox/training/models/Book.java

```diff
@@ -124,6 +134,7 @@ public String getPages() {
     }
 
     public void setPages(String pages) {
+        checkArgument(!Strings.isNullOrEmpty(pages));
         this.pages = pages;
```

Comment:
After checking if it is null or empty, you can check if the content of the page field is greater than 0.